apps/ocas/src/openc2_handler.erl

```diff
@@ -135,28 +128,29 @@ has_action(true, Req, State ) ->
     JsonMap = maps:get(json_map, State2),
     ActionBin = maps:get( <<"action">>, JsonMap ),
     lager:info("action bintext: ~p", [ActionBin] ),
-    { ActionValid, ActionValue } = actions:is_valid_action(ActionBin),
+    { ActionValid, {ActionModule, ActionFunction} } = actions:is_valid_action(ActionBin),
     State3 = maps:put(action_valid, ActionValid, State2),
-    State4 = maps:put(action, ActionValue, State3),
+    State4 = maps:put(action_module, ActionModule, State3),
+    State5 = maps:put(action_function, ActionFunction, State4),
 
     %% check if action is valid
-    check_valid_action( ActionValid, ActionValue, Req, State4 ).
+    check_valid_action( ActionValid, ActionModule, ActionFunction, Req, State5 ).
```

Comment:
Line 137 is too long:     check_valid_action( ActionValid, ActionModule, ActionFunction, Req, State5 )..